recipe/meta.yaml

```
@@ -29,14 +29,14 @@ requirements:
     - cython >=3.0.10,<4
     # Do not add any constraint on numpy here to use global pinning
     - numpy
-    - scipy >=1.13.0,<1.16.0
+    - scipy >=1.13.0
     - pandas >=1.4
     - patsy >=0.5.6
     - setuptools_scm >=8.0.0,<9
   run:
     - numpy <3,>=1.22.3
     - python
-    - scipy !=1.9.2,>=1.8
+    - scipy !=1.9.2,>=1.8,<1.16.0
```

Let's flip the order here
```suggestion
    - scipy >=1.8,!=1.9.2,<1.16.0
```
For more details see https://github.com/conda-forge/conda-smithy/issues/2343